src/main/java/org/tron/core/db2/core/SnapshotManager.java

```diff
@@ -187,6 +187,7 @@ public void shutdown() {
         TimeUnit.MILLISECONDS.sleep(10);
       } catch (InterruptedException e) {
         System.out.println(e.getMessage() + e);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1639&issues=AWalU-zulHK86Hco_1pp&open=AWalU-zulHK86Hco_1pp)

[ISSUE_KEY]: AWalU-zulHK86Hco_1pp